src/components/navigation-header/components/product-page-content/utils/get-nav-items.ts

```diff
@@ -0,0 +1,57 @@
+import { NavigationHeaderIcon } from 'components/navigation-header/types'
+import { NavItem } from './types'
+import { ProductData, RootDocsPath } from 'types/products'
+
+/**
+ * Given current product data,
+ * Return an array of NavItems to render in the top navigation bar.
+ *
+ * Note that these items are only shown on larger viewports.
+ * On smaller viewports, a separate mobile menu is shown, which
+ * does not use this exact function to generate nav items,
+ * so may not show the same set of items.
+ */
+export function getNavItems(currentProduct: ProductData): NavItem[] {
+	/**
+	 * Define a common set of base nav items
+	 */
+	const items: NavItem[] = [
+		{ type: 'link', label: 'Home', url: `/${currentProduct.slug}` },
+		{
+			type: 'submenu',
+			label: 'Documentation',
+			iconColorTheme: currentProduct.slug,
+			items: currentProduct.rootDocsPaths.map((rootDocsPath: RootDocsPath) => {
+				return {
+					icon: rootDocsPath.iconName as NavigationHeaderIcon,
+					label: rootDocsPath.name,
+					path: `/${currentProduct.slug}/${rootDocsPath.path}`,
+				}
+			}),
+		},
+		{
+			type: 'link',
+			label: 'Tutorials',
+			url: `/${currentProduct.slug}/tutorials`,
+		},
+		{
+			type: 'link',
+			label: 'Install',
+			url: `/${currentProduct.slug}/tutorials`,
+		},
+	]
+	/**
+	 * For Terraform, add a "Registry" item
+	 */
+	if (currentProduct.slug == 'terraform') {
+		items.push({
+			type: 'link',
+			label: 'Registry',
+			url: 'https://registry.terraform.io/',
+		})
```

Comment:
I think we want to open this link in a new tab since it's external to the site. That's how we're handling the link for this item in the Sidebar Resources section as well.

Reply:
Sweet, can do! ✅ Have updated to support an `openInNewTab` property here, and have used it 👍